test/feeds-atom.test.js

```diff
@@ -1,3 +1,7 @@
+/**
```

Comment:
Why not specify this env once in the config for these test files here https://github.com/Seneca-CDOT/telescope/blob/master/test/jest.config.js?

Reply:
Thanks for the suggestion! I added the env to `jest.config.js` and now all the tests pass.